### PR TITLE
Getting started page, one more time

### LINKED ISSUE #4315: Feedback for Getting Started page
Feedback for the new Getting Started page:
https://preliminary.istio.io/docs/setup/kubernetes/getting-started/

Assuming that this is the entry point for any new users getting started with Istio, I have some feedback for this page.

1. This page should not be verbose or provide users with too many options. This should be opinionated happy path install that targets majority of our users... installing and trying out istio on an existing vanilla k8s cluster.
2. The page starts off with too many distractions and complexities. Information about CNI, multi cluster, mesh expansion and sidecar injection guide before the user even downloads Istio is not necessary and intimidates the users - get them started quickly.
3. The page only has one main section - “Downloading the release” and then abruptly ends. What should they do next? Point them to, or merge content from “Quick Start Evaluation Install”. Again, if we want majority of new users to use the demo profile, then just have them install it on this page instead of moving them around.
4. After the Download / Install section, the user should be guided to perform some easy and “high value” tasks. For example: Request Routing and Visualizing Your Mesh
5. The “See also” section is mostly content that is completely irrelevant to the user at this step.

### PR DESCRIPTION
**Pull request by frankbu**
Fixes #4315

### CODE REVIEW

File: content/docs/setup/kubernetes/getting-started/index.md [hunk 1]
```diff
@@ -2,83 +2,41 @@
...
-
-- [Supported platform installation](/docs/setup/kubernetes/install/platform/):
-   This option is ideal if your platform provides native support for Istio-enabled clusters
-   with a [configuration profile](/docs/setup/kubernetes/additional-setup/config-profiles/)
-   corresponding to your intended use.
-
-After choosing an option and installing Istio on your cluster, you can deploy
-your own application or experiment with some of our [tasks](/docs/tasks/) and [examples](/docs/examples/).
-
 {{< tip >}}
```
Comment (rvennam) [inline content/docs/setup/kubernetes/getting-started/index.md]: Starting the page with a tip section looks a bit strange. What do you think about removing the tip and adding the sentence "Istio can also be installed and customized in many [other ways](/docs/setup/), depending on your platform and intended use." to the end of the "If you’re new to Istio" sentence.?

Comment (frankbu) [reply content/docs/setup/kubernetes/getting-started/index.md]: good idea

File: content/docs/setup/kubernetes/_index.md [hunk 2]
```diff
@@ -15,12 +17,88 @@ content_above: true
...
+- [Custom installation with Helm](/docs/setup/kubernetes/install/helm/):
+   This option is ideal to install Istio for production use or for performance evaluation.
+
+- [Supported platform installation](/docs/setup/kubernetes/install/platform/):
+   This option is ideal if your platform provides native support for Istio-enabled clusters
+   with a [configuration profile](/docs/setup/kubernetes/additional-setup/config-profiles/)
+   corresponding to your intended use.
+
+After choosing an option and installing Istio on your cluster, you can deploy
+your own application or experiment with some of our [tasks](/docs/tasks/) and [examples](/docs/examples/).
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/_index.md]: applications

File: content/docs/setup/kubernetes/_index.md [hunk 3]
```diff
@@ -15,12 +17,88 @@ content_above: true
...
+
+{{< tip >}}
+If you're running your own application, make sure to
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/_index.md]: applications

File: content/docs/setup/kubernetes/_index.md [hunk 4]
```diff
@@ -15,12 +17,88 @@ content_above: true
...
+- To install using Istio's Container Network Interface
+(CNI) plugin, visit our [CNI guide](/docs/setup/kubernetes/additional-setup/cni/).
+
+- To perform a multicluster setup, visit our
+[multicluster installation documents](/docs/setup/kubernetes/install/multicluster/).
+
+- To expand your existing mesh with additional containers or VMs not running on
+your mesh's Kubernetes cluster, follow our [mesh expansion guide](/docs/setup/kubernetes/additional-setup/mesh-expansion/).
+
+- To add services requires detailed understanding of sidecar injection. Visit our
```
Comment (geeknoid) [inline content/docs/setup/kubernetes/_index.md]: a detailed

**Pull request merged**